src/Search/Console.php

```diff
@@ -54,6 +58,28 @@
         ],
     ];
 
+    public function queuePushCommand(InputInterface $input, OutputInterface $output)
+    {
+        $helper = new QuestionHelper();
+        // Get the type.
+        $choice = new ChoiceQuestion('<question>Which type would you like to import</question>', [
+            'research-article',
+            'blog-articles',
+            'collection',
+        ], 0);
+        $type = $helper->ask($input, $output, $choice);
+        // Ge the Id.
+        $choice = new Question('<question>Whats the ID of the item to import: </question>');
+        $id = $helper->ask($input, $output, $choice);
+        // Create queue item.
+        $item = new QueueItemMock($type, $id);
```

Comment:
given we are now using this for real items, `Mock` can be misleading; the other implementation is `SqsMessage` so they could become `BusSqsMessage` and `InternalSqsMessage` respectively?